dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -97,24 +101,30 @@
 
     private final ToolDAO toolDAO;
     private final WorkflowDAO workflowDAO;
+    private final OrganizationDAO organizationDAO;
+    private final CollectionDAO collectionDAO;
     private final DockstoreWebserviceConfiguration config;
 
     public MetadataResource(SessionFactory sessionFactory, DockstoreWebserviceConfiguration config) {
         this.toolDAO = new ToolDAO(sessionFactory);
         this.workflowDAO = new WorkflowDAO(sessionFactory);
+        this.organizationDAO = new OrganizationDAO(sessionFactory);
+        this.collectionDAO = new CollectionDAO(sessionFactory);
         this.config = config;
     }
 
     @GET
     @Timed
     @UnitOfWork(readOnly = true)
     @Path("sitemap")
-    @Operation(summary = "List all published workflow and tool paths", description = "List all published workflow and tool paths, NO authentication")
-    @ApiOperation(value = "List all published workflow and tool paths.", notes = "NO authentication")
+    @Operation(summary = "List all available workflow, tool, organization, and collection paths", description = "List all available workflow, tool, organization, and collection paths. Available means published for tools/workflows, and approved for organizations and their respective collections. NO authentication")
+    @ApiOperation(value = "List all available workflow, tool, organization, and collection paths.", notes = "NO authentication")
```

Comment:
Probably best to keep OpenAPI 3 and Swagger the same.  

Also, even though we haven't really been consistent, "No authentication" probably isn't required since it's obvious in the swagger-ui due to the absence of the lock icon (because there's no `@Auth` and/or `@Authorization`).

Others can chime in on what's preferred and should be made standard

Reply:
We use the phrase "No authentication" all over the place. I agree we can/should drop it, but suggest a separate ticket/PR to clean them all up.

Reply:
Just for clarification about my first point.  I expect swagger.yaml's summary to match the openapi.yaml's summary.  I expect the swagger.yaml's description to match the openapi.yaml's description

Reply:
Ohh okay, I kept the swagger description as "No authentication" just because that seemed to be the format the other endpoints have.

Reply:
Even further clarification (since there's 2 openapi.yaml).  I expect all 3 of their descriptions to match and all 3 of their summaries to match.

Reply:
To elaborate on a conversation we had locally. 
One of the openapi3 descriptions is auto-converted from 2 (and isn't really used for anything on our end as far as I know, while it is used by the FAIRshake people IIRC).
One of the openapi3 descriptions is generated via a Maven plugin and is hooked up to tests and is intended to demonstrate using only the OpenAPI3 annotations for new endpoints going forward. 

That said, I think as Charles mentioned, I think some of this work can be split out into a separate ticket, but I think we can trust you to resolve this appropriately for now.